jobs/delorean/jenkinsfiles/discovery/rc/github/Jenkinsfile

```diff
@@ -124,7 +124,8 @@ node {
                     productImages = rhRegistries.collect { registry ->
                         registryProjectIDs.collect { registryID ->
                             println "Checking for registry: ${registry}, registryID: ${registryID}"
-                            sh(returnStdout: true, script: "find . -name \'*.y*ml\' -exec grep -oP \'${registry}/${registryID}.*/[^[:blank:]].*\' {} \\; | sort | uniq").replace('"', '').split('\n')
+                            // Ignore fuse-online-upgrade as it contains an image with a typo (registry.redhat.io/fuse7/fuse-ignite-uprade)
```

Comment:
hmm that's annoying. Have we commented or PRd a change for this upstream?

Reply:
I've created an issue against that repo. Looks like it's an old file leftover before the operator existed. Not sure what their plan is with that yet. https://github.com/syndesisio/fuse-online-install/issues/131